components/core/src/clp/ffi/ir_stream/UnstructuredIrDeserializerImpl.cpp

```
@@ -0,0 +1,300 @@
+
+template <ir::EncodedVariableTypeReq encoded_variable_t>
+auto UnstructuredIrDeserializerImpl<encoded_variable_t>::get_next_ir_unit_type(
+        ReaderInterface& reader
+) -> ystdlib::error_handling::Result<std::pair<IrUnitType, encoded_tag_t>> {
+    if (false == m_pending_schema_insertions.empty()) {
+        return std::pair{IrUnitType::SchemaTreeNodeInsertion, encoded_tag_t{0}};
+    }
+
+    auto const tag_result{deserialize_tag(reader)};
+    if (tag_result.has_error()) {
+        return tag_result.error();
```

Cppcheck (2.20.0)

[style] The function 'get_children_ids' is never used.

(unusedFunction)